include/openssl/ossl_typ.h

```diff
@@ -203,6 +203,16 @@ typedef long ossl_intmax_t;
 typedef unsigned long ossl_uintmax_t;
 #endif
 
+/*
+ * Use this inside a union with the field that needs to be aligned to a
+ * reasonable boundary for the platform.  The most pessimistic alignment
+ * of the listed types will be used by the compiler.
+ */
+# define OSSL_UNION_ALIGN       \
+    double align;               \
+    uintmax_t align_int;        \
```

Comment:
Does this need to be `ossl_uintmax_t` ?

Reply:
Yes :(

Your English was okay the first time around BTW :)


FIxed.